lib/Db/ImageMapper.php

```diff
@@ -119,10 +132,12 @@ public function findImagesFromPerson(string $userId, string $name, int $model):
 			->andWhere($qb->expr()->eq('model', $qb->createParameter('model')))
 			->andWhere($qb->expr()->eq('is_processed', $qb->createParameter('is_processed')))
 			->andWhere($qb->expr()->like($qb->func()->lower('p.name'), $qb->createParameter('query')));
-			$params['user'] = $userId;
-			$params['model'] = $model;
-			$params['is_processed'] = True;
-			$params['query'] = '%' . $this->db->escapeLikeParameter(strtolower($name)) . '%';
+
+		$params = array();
+		$params['user'] = $userId;
+		$params['model'] = $model;
+		$params['is_processed'] = True;
+		$params['query'] = '%' . $this->db->escapeLikeParameter(strtolower($name)) . '%';
```

Comment:
where is this `$name` from?

Reply:
MMm.. :confused:
The $name of argument is compared with person.name on:
https://github.com/matiasdelellis/facerecognition/pull/49/files/fb559becac0f63fe1a87c5ce8c3076ed7042a993#diff-5dc310fdcd7b76f8112d2d7b4ed94866R134

...with the $param['query'] using an 'LIKE' sentence in that line...

Reply:
OMG, sorry, Github collapsed origibal method and I was looking at wrong method signature...just ignore, I am stupid:)